js/src/launchAsync.ts

```diff
@@ -154,3 +160,7 @@ export function launchAsync(token: string, subdomain: string, content: Content,
         document.head.appendChild(noscroll);
     });
 }
+
+export function close() {
+    window.postMessage('ImmersiveReader-Exit', '*');
+}
```

Comment:
We need to update the documentation for this.
In addition, do we need to listen to ImmersiveReader-Exit messages from the WebApp and relay them to the hosting app when the exit button exists and is pressed by the user?

Reply:
In addition, perhaps the close() method should also remove the iframe and reset the internal state (call reset())
?

---
In reply to: [327718990](https://github.com/microsoft/immersive-reader-sdk/pull/97#discussion_r327718990) [](ancestors = 327718990)

Reply:
Going to send a PR for updating documentation. 
Line 94 is listening for "ImmersiveReader-Exit". I could move all of the logic inside of that conditional into the close() function but I think that it has the same functionality.

Reply:
https://github.com/MicrosoftDocs/azure-docs-pr/pull/89590